### PR TITLE
Tidy up home page templates (Fixes #10871)

### LINKED ISSUE #10871: Tidy up home page templates
## Description

We currently have 5 different home page templates:

- mozorg
  - home
    - `home-de.html` (German home page)
    - `home-en.html` (Fallback English home page)
    - `home-fr.html` (French home page)
    - `home.html` (Rest of world home page)
  - `contentful-homepage.html` (Contentful English home page)

Some tidy up work that might make updating the home pages easier:

1. The template `contentful-homepage.html` is in a different directory to all the other templates. Moving it into `mozorg/home/` seems like a no brainer, as I nearly always make changes to `home-en.html` and then wonder why I can't see any updates.
2. The template `home-en.html` is not easy to view locally without making code changes, and copy/pasting changes in `contentful-home-page.html` over to `home-en.html` seems like a chore people are going to forget over time. I wonder if it makes sense to rename the Contentful home page to `mozorg/home/home-en.html`, and then fallback to showing `home.html` (the rest-of-world home page) should there be a problem with Contentful. This might also nudge us to make updates that also benefit the rest of world more? Essentially we'd be back down to 4 templates.

---

#### Success Criteria
- [ ] Home page templates are updated depending on consensus to the above.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Moves/renames `mozorg/contentful-homepage.html` to `mozorg/home/hone-en.html`.
- Falls back to `mozorg/home/home.html` if Contentful errors.
- Creates a new `mozorg/home/base.html` template that all home pages extend from, so we avoid repeating metadata.
- Renames `home-2018` CSS bundles to `home-eu` since it relates only to the DE/FR home pages.
- Removes old MR1 promo from `mozorg/home/home.html`.
- Updates functional tests for home pages.

## Issue / Bugzilla link
#10871

## Testing
- http://localhost:8000/es-US/
- http://localhost:8000/de/
- http://localhost:8000/fr/
- http://localhost:8000/es-ES/

Successful test run: https://gitlab.com/mozmeao/www-config/-/pipelines/424328544

### CODE REVIEW

**Review by maureenlholland — APPROVED** [review]
LGTM 🧹 
One question re: `home-en` name now that I see it in practice, not blocking

File: bedrock/mozorg/views.py [hunk 1]
```diff
@@ -122,27 +122,25 @@ def home_view(request):
...
+                template_name = "mozorg/home/home.html"
         else:
-            template_name = "mozorg/home/home-en.html"
-            ctx["page_content_cards"] = get_page_content_cards("home-en", "en-US")
+            template_name = "mozorg/home/home.html"
     elif locale == "de":
         if switch("contentful-homepage-de"):
             try:
-                template_name = "mozorg/contentful-homepage.html"
+                template_name = "mozorg/home/home-en.html"
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/mozorg/views.py:143]: I think this is a good suggestion, it will also match the CSS bundle name then 👍

**Comment by alexgibson** [comment]
@maureenlholland thanks for the review, updated. I also removed the (currently unused) experimental home page URL at https://www.mozilla.org/en-US/exp/, as even right now this is showing the wrong home page in production.

File: bedrock/firefox/redirects.py [hunk 2]
```diff
@@ -547,4 +547,6 @@ def firefox_channel(*args, **kwargs):
     redirect(r"^firefox/whatsnew/(india|africa|france|en|all|china)/?$", "firefox.whatsnew"),
     # issue 10703
     redirect(r"firefox/lockwise/?", "https://support.mozilla.org/kb/end-of-support-firefox-lockwise"),
+    # issue 10879
+    redirect(r"^/exp/?$", "mozorg.home"),
```
Comment (maureenlholland) [inline bedrock/firefox/redirects.py:551]: 👍

**Review by maureenlholland — APPROVED** [review]
🙌  nice catch on /exp too. r++

**Pull request merged**